types/setting.go

```diff
@@ -59,6 +59,7 @@ const (
 	SettingNameDisableSchedulingOnCordonedNode   = SettingName("disable-scheduling-on-cordoned-node")
 	SettingNameReplicaZoneSoftAntiAffinity       = SettingName("replica-zone-soft-anti-affinity")
 	SettingNameReplicaRebuildConcurrentLimit     = SettingName("replica-rebuild-concurrent-limit")
+	SettingNameVolumeAttachmentDeletion          = SettingName("volume-attachment-deletion")
```

Comment:
`SettingNameDeploymentPodVolumeRecoveryPolicy`
`deployment-pod-volume-recovery-policy`

Reply:
Or `Kubernetes Volume Attachment Recovery Policy`?

Reply:
I like the `Kubernetes Volume Attachment Recovery Policy` better then the pod one.

Reply:
Going with just `Volume Attachment Recovery Policy`